po/zh_CN.po

```diff
@@ -5801,7 +5810,7 @@ msgstr "入账日期"
 #: gnucash/gnome/gnc-plugin-page-register.c:5088
 #, c-format
 msgid "Checking splits in current register: %u of %u"
-msgstr ""
+msgstr "我们检查当前记录书中的拆分：％U，％U."
```

Comment:
I fear, format specifiers must not be uppercase.
Finally I got:
msgfmt -c --check-accelerators="_" --statistics po/zh_CN.po 
po/zh_CN.po:5813: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:7341: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:7497: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:16329: msgstr lacks the keyboard accelerator mark '_'
po/zh_CN.po:18778: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19827: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19832: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19837: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19842: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19847: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19852: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19868: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19873: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19878: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19907: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19912: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19927: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:19932: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20019: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20130: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20150: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20170: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20272: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20296: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20442: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20458: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20483: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20497: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20516: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:20521: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:21153: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:21611: format specifications in 'msgid' and 'msgstr' are not equivalent
po/zh_CN.po:26266: format specifications in 'msgid' and 'msgstr' are not equivalent
po/zh_CN.po:27436: format specifications in 'msgid' and 'msgstr' are not equivalent
po/zh_CN.po:30076: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:30081: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:30087: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:30092: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:30529: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:30556: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:30561: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:30566: number of format specifications in 'msgid' and 'msgstr' does not match
po/zh_CN.po:30571: number of format specifications in 'msgid' and 'msgstr' does not match
msgfmt: found 43 fatal errors
4075 translated messages, 1285 fuzzy translations, 119 untranslated messages.
``